suite-native/module-onboarding/src/screens/UninitializedDeviceLandingScreen.tsx

```diff
@@ -0,0 +1,118 @@
+    (_, { hasDeviceFirmwareInstalled }) => ({
+        width: '100%',
+        height: hasDeviceFirmwareInstalled ? 280 : 360,
+        maxHeight: (getScreenHeight() / 3) * 2,
+        alignItems: 'center',
+    }),
+);
+
+const trezorModelImageMap = {
+    [DeviceModelInternal.T2B1]: require('../assets/trezorSafe3.png'),
+    [DeviceModelInternal.T3B1]: require('../assets/trezorSafe3.png'),
+    [DeviceModelInternal.T3T1]: require('../assets/trezorSafe5.png'),
+} as const satisfies Record<SetupSupportingDeviceModel, string>;
+
+const UninitializedDeviceLandingScreenContent = () => {
+    const { applyStyle } = useNativeStyles();
+    const deviceModel = useSelector(selectDeviceModel) as SetupSupportingDeviceModel;
+    const hasDeviceFirmwareInstalled = useSelector(selectHasDeviceFirmwareInstalled);
+
+    if (!deviceModel) {
+        return null;
+    }
+
+    return (
+        <VStack spacing="sp32">
+            {hasDeviceFirmwareInstalled ? (
+                <TitleHeader
+                    title={
+                        <Translation id="moduleOnboarding.uninitializedDeviceLandingScreen.firmware.title" />
+                    }
+                    titleVariant="titleMedium"
+                    subtitle={
+                        <Translation id="moduleOnboarding.uninitializedDeviceLandingScreen.firmware.subtitle" />
+                    }
+                />
+            ) : (
+                <Box alignItems="center">
+                    <Text variant="titleMedium" textAlign="center" style={{ letterSpacing: -0.5 }}>
+                        <Translation id="moduleOnboarding.uninitializedDeviceLandingScreen.noFirmware.title" />
+                    </Text>
+                    <HeaderUnderlineSvg />
+                </Box>
+            )}
+            <Image
+                source={trezorModelImageMap[deviceModel]}
+                contentFit="contain"
+                style={applyStyle(trezorImageStyle, {
+                    hasDeviceFirmwareInstalled,
+                })}
+            />
+        </VStack>
+    );
+};
+
+export const UninitializedDeviceLandingScreen = () => {
+    const { showToast } = useToast();
+    const hasDeviceFirmwareInstalled = useSelector(selectHasDeviceFirmwareInstalled);
+
+    const handleConfirmButtonPress = () => {
+        // TODO: navigate to next onboarding screen based on firmware status
+        showToast({ variant: 'warning', message: 'TODO: implement next screen' });
+    };
+
+    const handleDeclineButtonPress = () => {
+        // TODO: navigate to screen where user can see more info regarding tampered devices.
+        showToast({ variant: 'warning', message: 'TODO: implement next screen' });
+    };
+
+    return (
+        // TODO: add handling of close button event
+        <Screen header={<ScreenHeader closeActionType="close" />}>
+            <VStack justifyContent="space-between" flex={1} paddingVertical="sp16">
```

Comment:
I believe you want to use this:
```suggestion
            <VStack justifyContent="space-between" flex={1} paddingTop="sp16">
```

Reply:
fixed 68cc994372